app/directives/bucket-page-header-card/bucket-page-header-card.html

```diff
@@ -8,12 +8,12 @@
 
   <md-card-content class="bucket-page__description">
     <div layout="row" layout-align="start center" ng-if="bucket.target > 0">
-      <img ng-src="./img/bucket_empty.svg" alt="bucket" class="bucket-page-header-card__icon"/>
+      <img src="./img/bucket_empty.svg" alt="bucket" class="bucket-page-header-card__icon"/>
```

Comment:
are these `ng-XXX` mods, bug fixes for something unrelated?

Reply:
these were just bugs... they were "working" ok but ultimately were not being used right. `ng-src` is intended to use when the src is being interpolated from data.